src/style.css

```diff
@@ -2,0 +1,90 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+html {
+  font-family: 'Avenir';
+  font-weight: 300;
+}
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 20px;
+}
+
+.inputs__bigInput {
+  width: 100%;
+  font-family: 'Avenir';
```

Comment:
Consider adding a generic fallback font to the 'Avenir' font-family for the .inputs__bigInput class.